src/js/view/controls/components/menu/category-button.ts

```diff
@@ -32,9 +37,11 @@ const categoryButton = (menu) => {
     buttonElement.setAttribute('aria-expanded', 'false');
     buttonElement.setAttribute('aria-haspopup', 'true');
     buttonElement.setAttribute('aria-controls', menu.el.id);
+
     if (!('ontouchstart' in window)) {
-        menuCategoryButton.tooltip = SimpleTooltip(buttonElement, name, title);
+        (menuCategoryButton as CategoryButton).tooltip = SimpleTooltip(buttonElement, name, title);
```

Comment:
can you do `const menuCategoryButton: CategoryButton =...` above instead?

Reply:
ButtonCategoryButton is a member of Button, so this is a safe cast. That being said, I like your suggestion more because the return type will always be CategoryButton